### ISSUE TITLE
Error set process stage with conditional BPF

### ISSUE THREAD

**Issue by WestJackson** [demianrasko/Dynamics-365-Workflow-Tools#92]
Hello there,

I saw some posts that I think were similar, but wanted to make you aware of this issue. It has to do with Dynamics new conditional BPFs.

I'm using workflow tools to set the process stage of a BPF in the background. Currently I am using a conditional/branched BPF (see screenshot).
![image](https://user-images.githubusercontent.com/6888169/42703533-b748b562-8681-11e8-95fc-dc84cb88b21a.png)

The normal path:
Qualify --> Level 5 --> Level 4 --> level 3 -->...

The conditional branch/path:
Qualify --> Fast Track Prep --> Level 3--> ...

Note how they converge back at level 3. 

I am currently trying to use the Set Process Stage to move the Opportunity to a level 3 when I am at the "Fast Track Prep" stage where an opportunity meets the fast track condition.

### Expected behavior and actual behavior.
**Expected behavior:** 
Move next stage to a level 3 and keep BPF under the conditional path. Would look like this:
![image](https://user-images.githubusercontent.com/6888169/42703260-ef02280e-8680-11e8-8e24-5ea70dc63c03.png)

**Actual Behavior:**
Opportunity moves to a level 3, BUT it changes the active path of the Opportunity even though the condition is still satisfied. For example, this Opportunity is still supposed to be on the fast track path. After using the 'set process stage' to a level 3, it now has changed the active path back to the normal un-conditional path (confirmed with Xrm.Page.data.process.getActivePath())

![image](https://user-images.githubusercontent.com/6888169/42703326-1f2f9c1e-8681-11e8-8f1e-6c0aa7af9adc.png)

If you try to ever 'move previous' the system runs into an error (probably because it thinks it should be on the conditional path but is not).

### Steps to reproduce the problem (screenshots are really usefull)
Basically have a BPF with a conditional path. While on the conditional path, use workflowtools' Set Process stage to a stage that is where the conditional and normal paths converge (level 3 in this example). 

### CRM Version - online/onpremise - WorkflowTools version
msdyncrmWorkflowTools (1.0.52.1)
CRM online, Version 1612 (8.2.2.2060)

Any thoughts on this? Am I doing something wrong or is this just a bug?
Thanks!

**Comment by demianrasko** [comment]
I think there was an issue because I was using the old CRM 2013 way to move the phases stages. Now is updated to a new multi process way (and conditional), so maybe now it works fine.

Solved today. will be deployed in the next version 1.0.53.0 during this weekend. 
Thanks!
